ChemistryLib/PhreeqcInterfaceData/Solution.h

```diff
@@ -0,0 +1,83 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <boost/optional/optional.hpp>
+#include <fstream>
```

Comment:
could you please try to include <iosfwd> in the header files?

Reply:
Changed.